### ISSUE TITLE
Error output display incorrect for files with windows-style line breaks

### ISSUE THREAD

**Issue by misterflippy** [prantlf/jsonlint#1]
The error location output is incorrect for files with windows-style line breaks. (\r\n)

Example file: [test.zip](https://github.com/prantlf/jsonlint/files/3356865/test.zip)

This file produces error output of:
```
Parse error on line 4, column 3:
} } "bar":
---------------------^
Unexpected token }
```

Expected output: The output should include the referenced bracket ("}") and the arrow should point to it like this:
```
Parse error on line 4, column 3:
...": {    "bar":  }}
-------------------^
Unexpected token }
```

**Comment by prantlf** [comment]
Thanks a lot for reporting this and for the right fix! I merged and released it in [v8.0.1](https://github.com/prantlf/jsonlint/releases/tag/v8.0.1). I included  your example as a new test case.

**Comment by misterflippy** [comment]
Glad to help! Thanks for the library :)
